Makefile.rules

```diff
@@ -1303,18 +1307,11 @@ $(srcdir)/src/config.h.in: $(configure_deps)
 	touch $@
 endif
 
-GNUmakefile: $(srcdir)/GNUmakefile.in config.status
-	$(SHELL) ./config.status $@
+%: $(srcdir)/%.in config.status
+	@$(SHELL) ./config.status $@
 
 libtool: config.status
-	$(SHELL) ./config.status $@
-
-doc/make_doc: $(srcdir)/doc/make_doc.in config.status
-	$(SHELL) ./config.status $@
-
-gac: $(srcdir)/cnf/gac.in config.status
-	$(SHELL) ./config.status $@
-
+	@$(SHELL) ./config.status $@
```

Comment:
Ahhh wait this is missing some of the fixes I made later on. Will update this PR